packages/frontend/editor-ui/src/components/WorkflowSelectorParameterInput/WorkflowSelectorParameterInput.vue

```diff
@@ -191,7 +198,27 @@ function openWorkflow() {
 	window.open(getWorkflowUrl(props.modelValue.value?.toString() ?? ''), '_blank');
 }
 
+async function refreshCachedWorkflow() {
+	if (!props.modelValue || props.modelValue.mode !== 'list' || !props.modelValue.value) {
```

Comment:
Would checking just for `props.modelValue.value` be enough here?

```suggestion
	if (!props.modelValue.value || props.modelValue.mode !== 'list') {
```

Reply:
I was confident that `props.modelValue` can be undefined, but maybe that's not the case.